recipes/cargo/cargo.inc

```diff
@@ -14,8 +14,12 @@ DEPENDS = "openssl zlib libgit2 curl ca-certificates libssh2"
 SRC_URI = "\
 	git://github.com/rust-lang/cargo.git;protocol=https;name=cargo \
 	git://github.com/rust-lang/rust-installer.git;protocol=https;name=rust-installer;destsuffix=git/src/rust-installer \
+	http://static-rust-lang-org.s3.amazonaws.com/cargo-dist/2015-04-02/cargo-nightly-x86_64-unknown-linux-gnu.tar.gz \
```

Comment:
Having this as an option makes sense, but I'd really rather it not be mandatory.

Could you add a PACKAGECONFIG setting that allows us to chose between using the rustc-snapshot & depending on a local rustc build?